test-basic/plan-builder-generated.js

```diff
@@ -1732,6 +1744,36 @@ describe('plan builder', function() {
             done();
         }).catch(done);
     }); 
+    it('vec.base64Decode#1', function(done) {
+        if(serverConfiguration.serverVersion < 12) {
+            this.skip();
+        }
+        testPlan([p.xs.string("abc")], p.vec.base64Decode(p.col("1")))
```

Comment:
What is this test actually doing? It looks like it's making sure that the response has "null" in it somewhere - how do we know that's ensuring that base64Decode works? 

I think the tests that @BillFarber  added for the vector operations for the Java Client would be much better - i.e. create a plan that binds new columns with the output of a bunch of vector operations, and verify that the bound columns have the expected values.

Reply:
This was auto-generated. I added the skip block since it fails on server versions less than 12.